CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
+* Websocket SSL `teardown` also tears down underlying TCP.
 * Update WebSocket examples to set TLS SNI.
 
+API Changes:
+
+* Previously, `teardown` and `async_teardown` of a websocket connection over SSL
+  would only shut down the SSL layer, leaving the TCP layer connected. Now the 
+  SSL teardown will tear down both the SSL and TCP layers, cleanly shutting down 
+  the TCP connection and closing the socket.
+  In the highly unlikely scenario where the client expects the TCP socket to remain 
```

Comment:
This is non-sterile language and also making a subjective judgement ("highly unlikely")